daemon/api_snaps.go

```diff
@@ -607,6 +607,7 @@ func snapUpdateMany(inst *snapInstruction, st *state.State) (*snapInstructionRes
 	// TODO: use a per-request context
 	updated, tasksets, err := snapstateUpdateMany(context.TODO(), st, inst.Snaps, inst.userID, nil)
 	if err != nil {
+		// XXX: restore validation-sets from history?
```

Comment:
this is necessary if we set IsRefreshOfAllSnap above?

Reply:
Indeed, done.